#IntensityPoint2D 单个点的强度
float64 x
float64 y
float64 intensity